src/components/Header.tsx

```diff
@@ -0,0 +1,49 @@
+import { FormEvent, useState } from 'react';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  todos: Todo[],
+  addTodo: (title: string) => void,
+  isInputDisabled: boolean,
+};
+
+export const Header: React.FC<Props> = ({
+  todos, addTodo, isInputDisabled,
+}) => {
+  const [title, setTitle] = useState('');
+
+  const handleInputChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setTitle(event.target.value);
+  };
+
+  const handleFormSubmit = (event: FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    addTodo(title);
+    setTitle('');
+  };
+
+  return (
+    <header className="todoapp__header">
+
+      {todos.length > 0
+      && (
```

Comment:
You already had a similar comment, so you need to correct all places in the code, not just in the place of the comment
```suggestion
      {todos.length > 0 && (
```

Reply:
done